### PR TITLE
docs(nav): add active links hints on top nav

### LINKED ISSUE #1563: Active class on header links
Add an active class on a list-item of the header-nav in documentation website so that we always know which page is active (much like hover style)

### PR DESCRIPTION
**Pull request by vvo**
fixes #1563

active

### CODE REVIEW

File: docgen/plugins/helpers.js [hunk 1]
```diff
@@ -13,6 +13,12 @@ export default function helpers(filenames, metalsmith, cb) {
       const lang = opts && opts.lang;
       return highlight(src, lang);
     },
+    maybeActive(navPath, singlePathOrArrayOfPaths) {
+      const pathsToTest = [].concat(singlePathOrArrayOfPaths);
```
Comment (bobylito) [inline docgen/plugins/helpers.js]: Nice trick!

File: docgen/src/stylesheets/components/_dropdown.scss [hunk 2]
```diff
@@ -24,12 +24,12 @@ ul.simple-dropdown {
...
   text-align: left;
   border: 1px solid $sd-border-color;
-  border-radius: 3px;
+  border-radius: 0 0 3px 3px;
   background: $sd-background-color;
   border: none;
   padding: 0;
   box-shadow: 0 1px 4px 0 rgba(31,59,93,0.5);
-  animation: fadeInDown 0.6s cubic-bezier(0.23, 1, 0.32, 1);
+  // animation: fadeInDown 0.1s cubic-bezier(0.23, 1, 0.32, 1);
```
Comment (bobylito) [inline docgen/src/stylesheets/components/_dropdown.scss]: If it's commented maybe we should just remove it? Why remove it btw?

Comment (LukyVj) [inline docgen/src/stylesheets/components/_dropdown.scss]: you meant "Why commented it btw" ?

Comment (vvo) [inline docgen/src/stylesheets/components/_dropdown.scss]: Oops sorry, I will update this. 0.6s does feel too slow (you really feel the "wait") I think there are recommendations as for animation timings. Will have a look. Right now its instant and not too brutal

File: docgen/plugins/helpers.js (hunk 1 above)
Comment (vvo) [inline docgen/plugins/helpers.js]: I had to test this in console to actually enaure its doing what I want eheh but yes funn

Comment (bobylito) [inline docgen/plugins/helpers.js]: ❤️

File: docgen/src/stylesheets/components/_dropdown.scss (hunk 2 above)
Comment (bobylito) [inline docgen/src/stylesheets/components/_dropdown.scss]: Well I trust @LukyVj on this one :)

**Comment by vvo** [comment]
Added back the animation but a faster one

**Pull request merged**